recipe/meta.yaml

```diff
@@ -383,91 +300,139 @@ outputs:
           source ${PREFIX}/etc/conda/activate.d/{{ PKG_NAME }}_activate.sh
           test -n "${OMNISCI_ROOT_PATH+x}"
           ldd ${PREFIX}/lib/libDBEngine.so
-
     about:
       home: https://www.omnisci.com/
       license: Apache-2.0
       license_family: APACHE
-      license_file: LICENSE.md
+      license_file: omniscidb-common/share/doc/omnisci/LICENSE.md
       summary: The OmniSci database
-
       description: |
         OmniSciDB is an in-memory, column store, SQL relational database
         that was designed from the ground up to run on GPUs.
       doc_url: https://www.omnisci.com/docs/latest/
       dev_url: https://github.com/omnisci/omniscidb
```

Comment:
There is a redirect in the `dev_url` to the following `URL`
https://github.com/heavyai/heavydb